src/support/file.h

```diff
@@ -29,15 +29,15 @@
 
 namespace wasm {
 template <typename T>
-T read_file(const std::string &filename, bool debug);
+T read_file(const std::string &filename, bool binary, bool debug);
```

Comment:
It would have just 2 values though, and seem inconsistent with the next param after it, which is a bool?